xdsl/transforms/stencil_bufferize.py

```diff
@@ -213,6 +213,9 @@ class ApplyStoreFoldPattern(RewritePattern):
     def is_dest_safe(apply: ApplyOp, store: StoreOp) -> bool:
         # Check that the destination is not used between the apply and store.
         dest = store.field
+        start = dest.owner
+        if isinstance(start, Block):
+            start = cast(Operation, start.first_op)
```

Comment:
`start` is unused

Reply:
Sure, just trying to make this PR a dialect PR now. Redeleted unused start. Merge at your convenience